python/ray/tune/trainable/util.py

```diff
@@ -185,6 +185,15 @@ def get_checkpoints_paths(logdir):
         )
         return chkpt_df
 
+    @staticmethod
+    def get_remote_storage_path(
+        local_path: str, logdir: str, remote_checkpoint_dir: str
+    ) -> str:
+        """Converts a ``local_path`` to be based off of
+        ``remote_checkpoint_dir`` instead of ``logdir``."""
```

Comment:
We should mention the assumption that `logdir` is a prefix of `local_path` in this docstring.